### ISSUE TITLE
Add support for Visual Studio Code

### ISSUE THREAD

**Issue by buildersbrewery** [buildersbrewery/linden-scripting-language#13]
Add support for [Visual Studio Code](https://code.visualstudio.com).

* [ ] grammar
* [ ] highlighting
* [ ] snippets
* [ ] intellisense
* [ ] symbols
* [ ] theme
* [ ] settings
* [ ] bracket matching
* [ ] linting
* [ ] debugging

**Comment by dimentox** [comment]
Yup will get those as i get the details of how its supposed to work.. right now its been trial and error. 
1

**Comment by buildersbrewery** [comment]
@dimentox 
I hop to fix the grammar and test files within the nex few days. Feel free to play around with the snippets I have added.

**Comment by dimentox** [comment]
Awesome... Sorry i got busy LOL

On Wed, May 13, 2015 at 12:06 PM, buildersbrewery <notifications@github.com>
wrote:

> @dimentox <https://github.com/dimentox>
> I hop to fix the grammar and test files within the nex few days. Feel free
> to play around with the snippets I have added.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/buildersbrewery/linden-scripting-language/issues/13#issuecomment-101748395>
> .
>



-- 
-------------------------------------------------------------------------------------------------------------------------------
This email is a private and confidential communication. Any use of email
may be subject to the laws and regulations of the United States. You may
not Repost, Distribute nor reproduce any content of this message.
-------------------------------------------------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------------------------------
